### ISSUE TITLE
Allow user to submit extra columns

### ISSUE THREAD

**Issue by g-eoj** [h2oai/dai-deployment-templates#70]
Currently if you give the MOJO scorer extra columns that weren't used for training it ignores them. However for lamda deployment we throw an error:

```
ai.h2o.mojos.deploy.common.transform.ScoreRequestFormatException: Input fields don't match the Mojo
```

The behavior of lambda deployment should match local MOJO behavior.

**Comment by g-eoj** [comment]
Possible cause: https://github.com/h2oai/dai-deployment-templates/blob/master/common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/RequestChecker.java#L45

**Comment by osery** [comment]
Thanks for the report @g-eoj, you are correct. Just let me understand why that is a good idea and what is the expected behavior. Should we just allow additional columns or also allow fewer columns?
My concern is that if we get too permissive, we can easily hide mistakes such as small difference in a column name on the input (e.g. `name` vs. `Name`) silently resulting in ignoring the `Name` column and interpreting the `name` columns as all `N/A`s.

**Comment by osery** [comment]
I believe this has been fixed (now part of both 1.7 and LTS branch). Please reopen if this has not been addressed sufficiently.
